xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -609,4 +632,4 @@ function setGuestExtendedRights(self)
       }
     }
   }
-}
+}
```

Comment:
the last new line was removed. Was this intentional?

Reply:
My bad, I had trouble figuring out what this was about.
I understood why it happened (text mode edits in Linux) and fixed it.